### PR TITLE
CI test: Make sure FsharpSuite test failure stops and fails whole CI job

### PR DESCRIPTION
**Pull request by T-Gro**

### CODE REVIEW

**Comment by T-Gro** [comment]
Do not update, do not merge.
I want it to see fail first.

**Comment by T-Gro** [comment]
Now it fails as it should:

image


Next step, I am going to re-ignore the failing test, so that CI is green and this can get merged.

File: tests/fsharp/Compiler/Language/ByrefTests.fs [hunk 1]
```diff
@@ -267,7 +267,7 @@ let test () =
                 |]
 
     [<Test()>]
-    [<Ignore("Waiting for test to be aligned", Until = "2023-04-01 12:00:00Z")>]
+    //[<Ignore("Waiting for test to be aligned", Until = "2023-04-01 12:00:00Z")>]
```
Comment (T-Gro) [inline tests/fsharp/Compiler/Language/ByrefTests.fs:270]:
```suggestion
    [<Ignore("Waiting for test to be aligned", Until = "2023-04-01 12:00:00Z")>]
```

**Review by abonie — APPROVED** [review]

**Review by KevinRansom — APPROVED** [review]

**Pull request merged**